src/mbgl/map/transform.cpp

```diff
@@ -132,8 +129,8 @@ void Transform::easeTo(const CameraOptions& camera, const AnimationOptions& anim
     state.rotating = bearing != startBearing;
 
     startTransition(camera, animation, [=](double t) {
-        Point<double> framePoint = util::interpolate(startPoint, endPoint, t);
-        LatLng frameLatLng = Projection::unproject(framePoint, startScale);
+        LatLng frameLatLng(util::interpolate(startLatLng.latitude(), latLng.latitude(), t),
+                           util::interpolate(startLatLng.longitude(), latLng.longitude(), t));
```

Comment:
Interpolating latitude or longitude is a bit different than interpolating screen coordinates. In particular, a unit of latitude isn’t guaranteed to cover the same distance on screen as a unit of longitude, especially at higher latitudes, so I’d expect this change to lead to some distortion in the camera’s trajectory.

Reply:
@1ec5 
Good point. Thanks.